client/blocks/app-promo/style.scss

```diff
@@ -50,6 +39,32 @@
 	}
 }
 
+.app-promo .app-promo__paragraph {
+	margin-bottom: 0;
+	padding: 0 10px;
+}
+
+.app-promo .app-promo__jetpack-app-link {
+	color: var(--studio-jetpack-green-40);
+	font-weight: 700;
+	border-bottom: 1.5px solid var(--studio-jetpack-green-40);
+
+	&:hover {
+		color: var(--studio-jetpack-green-60);
+		border-bottom: none;
+	}
+}
+
+.app-promo .app-promo_jetpack-app-button {
+	color: #1e1e1e;
+
+	&:hover {
+		color: #1e1e1e;
```

Comment:
Any chance to use a color variable?

Reply:
Following the [discussion here](https://github.com/Automattic/wp-calypso/pull/68293#discussion_r981396905), I've gone ahead to replace `#1e1e1e` with `--color-text` (`#2c3338` by default).